Lib/test/test_readline.py

```diff
@@ -127,6 +127,13 @@ def test_init(self):
 input()
 print("History length:", readline.get_current_history_length())
 """
+    auto_history_script_with_null = """\
+import readline
+readline.set_auto_history({})
```

Comment:
Is readline history related to this issue? Does the test work if remove calls of `set_auto_history` and `get_current_history_length`?